classes/ValidFormBuilder/Password.php

```diff
@@ -135,6 +148,20 @@ public function __toHtml($submitted = false, $blnSimpleLayout = false, $blnLabel
 
             $this->setMeta("class", "vf__multifielditem");
 
+            if ($this->isRemovable()) {
+                $this->setMeta("class", "vf__removable");
+            }
+
+            //*** Add data-dynamic="original" or data-dynamic="clone" attributes to dynamic fields
```

Comment:
Please explain in short why you add the data attributes. There is no direct logic to derive the "why" from.